ports/winit/app.rs

```diff
@@ -141,8 +141,9 @@ impl App {
         std::mem::take(&mut *self.event_queue.borrow_mut())
     }
 
-    // This function decides whether the event should be handled during `run_forever`.
-    fn winit_event_to_servo_event(&self, event: winit::event::Event<'_, ServoEvent>) {
+    /// Processes the given winit Event, possibly converting it to an EmbedderEvent and
+    /// routing that to the App or relevant Window event queues.
+    fn winit_event_to_embedder_event(&self, event: winit::event::Event<'_, ServoEvent>) {
```

Comment:
As long as we are renaming this, this should probably have a better name, because currently it sounds like a conversion function. In particular, I think we always try to start function names with verbs. How do you feel about `queue_embedder_events_for_winit_event`?

Reply:
done in 8ced0b07128b018701100b5ec7d6409340271485